zed_components/src/zed_camera/src/zed_camera_component.cpp

```diff
@@ -93,6 +93,10 @@ ZedCamera::ZedCamera(const rclcpp::NodeOptions& options)
     if (!startCamera()) {
         exit(EXIT_FAILURE);
     }
+
+    // Dynamic parameters callback
+    // set_on_parameters_set_callback(std::bind(&ZedCamera::callback_paramChange, this, _1)); // deprecated
+    mParamChangeCallbackHandle = add_on_set_parameters_callback(std::bind(&ZedCamera::callback_paramChange, this, _1));
```

Comment:
Isn't this dangerous? If we move the callback initialization in the constructor the parameters are not yet defined causing unpredictable behaviors...

Reply:
This actually fixes #79. Maybe I didn't understand you correctly, but moving to the end of the constructor will actually mean that all parameters are set before allowing any external service or the ROS2 param command line tool to change them. This especially important for allowing ROS2 behind the scenes code to initialize parameters without having to go through this callback which doesn't handle their params update, and which is the root cause of #79.